cns/configuration/cns_config.json

```diff
@@ -5,6 +5,7 @@
         "HeartBeatIntervalInMins": 30,
         "RefreshIntervalInSecs": 15,
         "SnapshotIntervalInMins": 60,
+        "ConfigSnapshotIntervalInMins": 60,
```

Comment:
CNS can't be reconfigured without being restarted. What do we gain by re-emitting the config at an interval?

Reply:
@rbtr If we only emit this during start up, any CNS process that outlives the retention policy in Kusto (~90 days) won't be shown in the snapshots. Some constant logging of the config for multiple CNS nodes also gives us the ability to know what nodes in a cluster are active.

Reply:
I am convinced that we should re-emit the config, but I don't think that we should use that for liveness. And if we're not using it for liveness, and the retention is 90 days, 60 minutes seems a bit frequent.

Reply:
One of the usage scenarios for this is live site investigations where we want to determine the config with which CNS started. With that context, 60 minutes is not frequent.